packages/contracts/test/voting/token-voting.ts

```diff
@@ -1117,5 +1137,88 @@ describe('TokenVoting', function () {
         expect(await voting.canExecute(id)).to.equal(true);
       });
     });
+
+    describe('An edge case with >0% support and >=0% participation required in early execution mode', async () => {
```

Comment:
same here as the comment in addresslist.

Reply:
Fixed in f16d3e7becb9668bbd24b3ed065cdbee9af2b98f